### DISCUSSION TITLE
Datagrid: Make `sortable` configurable

### DISCUSSION THREAD

**Discussion by Dav1dde** [vmware-clarity/ng-clarity#309] [Ideas]
Currently if I have a server driven datagrid and want to enable filtering, I can set the `clrDgField` property, but this also forces me to use sorting. It implicitly sets `sortable` to true and there is no way for me to disable this behaviour.

I wish to using filtering but no sorting.

**Comment by chriscarpenter12** [comment]
And I have exact opposite use case where I use server driven data, and need to enable sorting, but leave filtering disabled on the column.

**Reply by kumar-tadepalli** [reply]
You can do that using clrDgSortBy
https://github.com/vmware/clarity/issues/2086#issuecomment-374338740

**Reply by chriscarpenter12** [reply]
@kumar-tadepalli Thanks for commenting! I totally missed that attribute as I thought it was for a local comparator function as I was doing server driven only, but it indeed works correctly.

```ts
<clr-dg-column [clrDgSortBy]="'createdAt'" [clrDgSortOrder]="sortDesc">Created At</clr-dg-column>
```
